doc/transform_methods/hansenlaw.rst

```diff
@@ -121,6 +121,19 @@ If you would like to access the Hansen-Law algorithm directly (to transform a
 right-side half-image), you can use :func:`abel.hansenlaw.hansenlaw_transform`.
 
 
+
+Tips
+----
+
+`hansenlaw` tends to perform better with images of large size :math:`n \gt 1001` pixel width. For smaller images the angular_integration (speed) profile may look better if sub-pixel sampling is used via: ::
+
+    transform_options=dict(dr=0.1)
```

Comment:
So, if dr is changed in `angular_integration_options`, then the result will change? I thought/think that the angular integration process should be independent of the number of bins used to calculate the integral. The `dr` passed in `transform_options` should correspond to the connection between pixel and "real world" units. The two should be unrelated.